src/App.js

```diff
@@ -1,5 +1,84 @@
+import { Message } from "./message.js";
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(Message.START);    
+    let randomNumber = MissionUtils.Random.pickUniqueNumbersInRange(1, 9, 3);
+    let strike = 0;
+
+    async function getUserAnswer() {
+      strike = 0;
+      let ball = 0;
+
+      const answer = await MissionUtils.Console.readLineAsync(Message.INPUT);
+      
+      if (answer.length === 3) { //입력한 값이 3자리인지 확인하는 유효성 검사
+
+        // 사용자 입력값을 백의 자리, 십의 자리, 일의 자리로 분류
+        const firstValue = Math.floor(answer / 100);
+        const secondValue = Math.floor((answer - (firstValue * 100)) / 10);
+        const thirdValue = Math.floor(answer - (firstValue * 100 + secondValue * 10))
+
+        // randomNumber 안에 firstValue, secondValue, thirdValue가 있는지 확인 > 낫싱&볼 검사
+        const firstValueInRandomNumber = randomNumber.includes(firstValue)
+        const secondValueInRandomNumber = randomNumber.includes(secondValue)
+        const thirdValueInRandomNumber = randomNumber.includes(thirdValue)
+        ball = firstValueInRandomNumber + secondValueInRandomNumber + thirdValueInRandomNumber;
+    
+        // randomNumber와 사용자 입력값 자리까지 비교 > 스트라이크 검사
+        if (ball) {
+          if (randomNumber[0] === firstValue) {
+            strike ++
+          }
+          if (randomNumber[1] === secondValue) {
+            strike ++
+          }
+          if (randomNumber[2] === thirdValue) {
+            strike ++
+          }
+        
+        // 검사 후 화면에 출력하기
+        MissionUtils.Console.print(`${strike}` + Message.STRIKE);
+          if (ball-strike) {
+            MissionUtils.Console.print(`${ball-strike}` + Message.BALL);
+          }
+          } else {
+              MissionUtils.Console.print(Message.NOTHING);
+          }    
```

Comment:
지금 제가 이해한 바로는 스트라이크가 0개이더라도 무조건 스트라이크 개수를 출력하는 것 같은데 스트라이크가 없을 경우는 볼만 출력하고 스트라이크와 볼 모두 없을 경우 낫싱을 출력하는 로직으로 알고 있어서요!!

```suggestion
        if (STRIKE !== 0 && BALL !== 0) {
        MissionUtils.Console.print(`${strike}` + Message.STRIKE + `${ball}` + Message.BALL);
      } else if (STRIKE !== 0) {
       MissionUtils.Console.print(`${strike}` + Message.STRIKE);
      } else if (BALL !== 0) {
        MissionUtils.Console.print(`${ball-strike}` + Message.BALL);
      } else {
        MissionUtils.Console.print(Message.NOTHING);
      }
```

이런 식으로 수정하는 건 어떨까요? 더 좋은 조건문이 있으면 추천해 주세요!